.travis.yml

```diff
@@ -11,7 +11,7 @@ install:
   - travis_retry composer install --prefer-dist --no-interaction --ignore-platform-reqs
 
 script:
-  - make ci
+  - make ci-with-coverage
```

Comment:
This won't work as expected, because the task in the Makefile doesn't use a container image that can collect coverage, you'd have to use our custom xdebug image for that.

I suggest to keep using `ci` task and the `after_success` Travis hook which runs a travis-native phpunit again for generating the coverage information. This will be faster because 

* Travis can generate the success event for GitHub earlier
* Travis doesn't need to download & start another docker image

As long as the tests don't fail because Travis's PHP environment differs too much from ours, we should use this slightly inconsistent "double strategy" IMHO. What do you think?